woocommerce/woocommerce-ios
# Updates tracks

Refs https://github.com/Automattic/Automattic-Tracks-iOS/pull/142

Updates to the latest version of tracks

To test:
Run the branch.
Visit the tracks dashboard. Look at live events for your user. Confirm that you see the updated device width and height properties.

Update release notes:

cc @aforcier 

@shiki could I trouble you for a review of this one?
